package.json

```diff
@@ -32,11 +32,13 @@
   },
   "scripts": {
     "build": "tsc && shx chmod +x dist/*.js",
+    "build:production": "npm run build && npm run build:set-release-stage",
+    "build:set-release-stage": "sed -i.bak 's/MCP_SERVER_RELEASE_STAGE = \"development\"/MCP_SERVER_RELEASE_STAGE = \"production\"/g' dist/common/config.js && rm dist/common/config.js.bak",
```

Comment:
this isnt ideal for anyone wanting to run in windows, maybe we could do this with node instead?